mcp-servers/oracle-sqlcl/helm/values.yaml

```diff
@@ -1,5 +1,5 @@
 image:
-  repository: quay.io/ecosystem-appeng/oracle-sqlcl
+  repository: quay.io/rhkp/oracle-sqlcl-mcp
```

Comment:
this should be reverted and also renamed this component to oracle-sqlcl. you need to rebase.